packages/browser-sdk/src/lib/react/useRoomConnection/index.ts

```diff
@@ -59,13 +58,12 @@ export function useRoomConnection(
     roomUrl: string,
     roomConnectionOptions = defaultRoomConnectionOptions,
 ): RoomConnectionRef {
-    const [store] = React.useState<Store>(() => {
-        if (roomConnectionOptions.localMedia) {
-            return roomConnectionOptions.localMedia.store;
```

Comment:
Are we dropping this store override? Was it being used anywhere? 🤔

Reply:
We don't need it any more. The reason this was here is that if you initialise local media hook first, and then the room connection hook, we need to use the same store. With the introduced Provider this is not an issue and we will have the same store anyways.